image.go

```diff
@@ -0,0 +1,9 @@
+package containerd
+
+import "github.com/containerd/containerd/images"
+
+type Image struct {
+	client *Client
+
+	i images.Image
+}
```

Comment:
No getters in Go. `Manifest`, `Config`, `Size`.

Although, we should be very careful what we consider size. This will be the packed size.